pyzbar/pyzbar.py

```diff
@@ -11,14 +11,15 @@
     zbar_image_create, zbar_image_destroy, zbar_image_set_format,
     zbar_image_set_size, zbar_image_set_data, zbar_scan_image,
     zbar_image_first_symbol, zbar_symbol_get_data,
+    zbar_symbol_get_loc_size, zbar_symbol_get_loc_x, zbar_symbol_get_loc_y,
     zbar_symbol_next, ZBarConfig, ZBarSymbol, EXTERNAL_DEPENDENCIES
 )
 
 __all__ = ['decode', 'EXTERNAL_DEPENDENCIES']
 
 
 # Results of reading a barcode
-Decoded = namedtuple('Decoded', ['data', 'type'])
+Decoded = namedtuple('Decoded', ['data', 'type', 'location'])
 
```

Comment:
A function to compute bounding boxes from scan locations (uses `operator.itemgetter`):

```
def bounding_box_of_locations(locations):
    x_values = list(map(itemgetter(0), locations))
    x_min, x_max = min(x_values), max(x_values)
    y_values = list(map(itemgetter(1), locations))
    y_min, y_max = min(y_values), max(y_values)
    return (Rect(x_min, y_min, x_max - x_min,  y_max - y_min))
```